showcase/package.json

```diff
@@ -36,6 +36,7 @@
     "test:a11y-report": "pnpm build:packages && ENABLE_A11Y_AUDIT=true ENABLE_A11Y_MIDDLEWARE_REPORTER=true ember test -f=\"Acceptance\""
   },
   "devDependencies": {
+    "@carbon/icons": "^11.70.0",
```

Comment:
[question] this should be declared in the `package/components/package.json` file, no?